wsd/wopi/WopiStorage.cpp

```diff
@@ -342,70 +342,61 @@ StorageBase::LockUpdateResult WopiStorage::updateLockState(const Authorization&
 
     try
     {
-        std::unique_ptr<Poco::Net::HTTPClientSession> psession(getHTTPClientSession(uriObject));
+        std::shared_ptr<http::Session> httpSession = getHttpSession(uriObject);
 
```

Comment:
`http::Session` isn't intended to be "shared" in the sense of multi-threading, rather we use `shared_ptr` because the reference to the instance is passed in the callback, when async requests are used. So `unique_ptr` wouldn't do.